### ISSUE TITLE
Allow to configure the target country

### ISSUE THREAD

**Issue by ziodave** [insideout10/wordlift-plugin#713]
Please describe the issue here:

We need to able to configure the target country.

Related to #682

**Comment by ziodave** [comment]
List of supported countries:

[cl.json.zip](https://github.com/insideout10/wordlift-plugin/files/1611909/cl.json.zip)

**Comment by ziodave** [comment]
* [ ] the country select should look like the language select:

![image](https://user-images.githubusercontent.com/11438/34952652-bb9c4348-fa1a-11e7-96de-ff82fc9705b5.png)

* [ ] the following message should be provided via PHP in order to support localization (and there's a typo to fix in _langugage_):

```$notices.html( 'The selected language is not supported in this country.</br>Please choose another country or langugage.' );```

* [ ] the settings form has a _save changes_ option, introducing ajax for only one option would be confusing, remove:

```
    // Post the validation request.
    wp.ajax.post(
      'wl_update_country_options',
      {
        lang: selectedLang,
        value: selectedCountry
      }
    )
    .done(function (data) {
      // Update the country select with new options
      $countrySelect.html(data);
    })
```

* [ ] is this used to load the list of countries? if so, this must be moved to the settings page fragment because settings are loaded at `admin_init` even when the page isn't loaded and the editor is just using the dashboard: loading it here slows down the dashboard:

```'country-codes' => json_encode( Wordlift_Countries::get_country_language_pairs() )```

* [ ] I don't understand this, language is required and country will be too:

```
 private function validate_country() {
         // Bail is for some reason the country and language are not set.
         if (
                 empty( $_POST['wl_general_settings']['site_language'] ) && // WPCS: CSRF, input var, sanitization ok.
                 empty( $_POST['wl_general_settings']['country_code'] ) // WPCS: CSRF, input var, sanitization ok.
         ) {
                 return;
         }
```

* [ ] I think it's better to change all the following occurrences, because we're publishing URLs and http://example.org/folder1/../folder2/file.ext looks quite ugly:

```plugin_dir_url( dirname( __FILE__ ) ) . '../admin/```...

to

```plugin_dir_url( dirname( dirname( __FILE__ ) ) ) . '../admin/```...

I don't though this is necessary because this is in the _src/admin/elements/class-wordlift-admin-select2-element.php_ file, why not do directly

```plugin_dir_url( dirname( __FILE__ ) ) . '```...

**Comment by ziodave** [comment]
Merged in #761 .
